end_to_end_tests/baseline_openapi_3.0.json

```diff
@@ -991,6 +991,20 @@
         }
       }
     },
```

Comment:
All of the files under `end_to_end_tests` can be ignored - they're just example data that's used by a test that runs the generator against a big spec file and checks for the expected output files.